x/commerciokyc/keeper/grpc_query_invite_test.go

```diff
@@ -58,3 +67,54 @@ func (suite *KeeperTestSuite) TestGRPCInvites() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestGRPCInvite() {
```

Comment:
Consider adding a test case where the store is empty (also in `TestGRPCInvites()`)